pack/parallel/pap.json

```diff
@@ -0,0 +1,94 @@
+[
+    {
+        "alternate_of": "02003",
+        "back_flavor": "Jenny Barnes has spent the majority of her young life in pursuit of creature comforts, fine dining, and the latest fashions. That all changed when she received a letter from her sister, Isabelle. In this letter, Isabelle confessed that mysterious forces were aligning against her and that she feared she may fall victim to some paranormal threat. It was the last letter Jenny received from her beloved sister. Jenny has since returned to the States to track down and investigate all occult occurrences she can find. Hardly a wilting flower, she has proven herself a crack shot as well as a fearless and clever investigator of the unknown. Until Isabelle's disappearance is explained, Jenny will never relent in her search.",
+        "back_text": "<b>Deck Size</b>: 30.\n<b>Deckbuilding Options</b>: Rogue cards ([rogue]) level 0-3, Neutral cards level 0-5, up to 10 other non-permanent [[Talent]] cards level 0-5.\n<b>Deckbuilding Requirements</b> (do not count toward deck size): Jenny's Twin .45s, Searching for Izzie, 1 random basic weakness.",
+        "code": "90084",
+        "deck_limit": 1,
+        "deck_options": [           
+            {
+                "faction": ["neutral"],
+                "level": { "min": 0, "max": 5 }
+            },
+            {
+                "faction": ["rogue"],
+                "level": { "min": 0, "max": 3 }
+            },
+            {
```

Comment:
The `deck_options` here can be simpler. a sufficient encoding for the talent filter can look like this:

```json
{
  "trait": ["talent"],
  "level": { "min": 0, "max": 5 },
  "permanent": false
}
```

This should work because the explicit `false` can (and should) be handled differently from `permanent` being undefined / null-ish in the data.

@zzorba should be able to corraborate this.

Reply:
I did try this, and while it did seem to work, it didn't work quite right. I temporarily modified the xp of Adaptable from 1 to 5, which should have put it outside of these deckbuilding rules, but it was still accessible:

![image](https://github.com/user-attachments/assets/f19f48b4-4e9d-4b95-9256-fdc76005bbe5)

Here's the rules I tested with:

```
            {
                "faction": ["neutral"],
                "level": { "min": 0, "max": 5 }
            },
            {
                "faction": ["rogue"],
                "level": { "min": 0, "max": 3 }
            },
            {
                "trait": ["talent"],
                "level": { "min": 0, "max": 5 },
                "permanent": false,
                "limit": 10
            }
```

Reply:
@vitaemachina I checked ArkhamCards and they handle it correctly, so there it should work:

https://github.com/zzorba/ArkhamCards/blob/d43dfe54300e2d064d2c9136856f90c8fe555716/src/lib/DeckValidation.ts#L546

In ArkhamDB, `option.permanent` is ignored unless it is truthy, so it makes sense why this would not work there:

https://github.com/Kamalisk/arkhamdb/blob/e780a86ecfe8cd0b81c39f85bc9093fe69414c2b/src/AppBundle/Resources/public/js/app.deck.js#L1656

https://github.com/Kamalisk/arkhamdb/blob/e780a86ecfe8cd0b81c39f85bc9093fe69414c2b/src/AppBundle/Helper/DeckValidationHelper.php#L239

It looks like an oversight to me, but not sure if it's worth the effort / coordination to fix this in exchange for less verbose / clearer deckbuilding. (@zzorba ?)

Reply:
Update: Arkham Cards now appears to have support for parallel jenny and looking at the data, the deck building implementation is the same as I suggested earlier.

Reply:
I've changed it to match what you recommended, and I've updated the date to match the release date of the comic.

Reply:
great, I will submit a patch to arkhamdb to adjust the deck validation